--- a/modules/juce_opengl/opengl/juce_OpenGLContext.cpp
+++ b/modules/juce_opengl/opengl/juce_OpenGLContext.cpp
@@ -977,7 +977,7 @@
                 const auto os = SystemStats::getOperatingSystemType();
 
                 if ((os & SystemStats::MacOSX) != 0)
-                    return (os != SystemStats::MacOSX && os < SystemStats::MacOSX_10_14);
+                    return (os != SystemStats::MacOSX && os < SystemStats::MacOSX_10_14) || os == SystemStats::MacOS_26;
 
                 return false;
             }();
